apps/03TicTacToe.js

```diff
@@ -24,24 +25,47 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+   return  (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) ||
+          (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) ||
+          (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn);
 }
 
+
 function verticalWin() {
-  // Your code here
+  return  (board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) ||
```

Comment:
Nice job short-handing your code with a `return` statement instead of an `if` statement.